ovn-tester/tests/cluster_density.py

```diff
@@ -17,9 +17,9 @@
 
 
 class ClusterDensity(ExtCmd):
-    def __init__(self, config, central_node, worker_nodes, global_cfg):
+    def __init__(self, config, central_nodes, worker_nodes, global_cfg):
```

Comment:
The only reason why we pass central_nodes and worker_nodes to all test constructors is because we need them in the constructor of ExtCmd/ExtCmdUnit.  We could instead pass the `Cluster` lists instead.

That also has the advantage that we don't need to pass the `clusters` argument to the `test.run()`function.  We already have it in the base class.

This applies to all test classes in `ovn-tester/tests/`

Reply:
@dceara ack, I will fix it